cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -930,6 +930,7 @@ constexpr bool is_split_decode()
  * @param num_rows Maximum number of rows to read
  * @param page_mask Boolean vector indicating which pages need to be decoded
  * @param initial_str_offsets Vector to store the initial offsets for large nested string cols
+ * @param page_string_offset_indices Device span of page string offset indices
```

Comment:
Maybe 'Device span of offset starts, indexed by page'?  My initial reading of this was "these are the offsets for the strings in this page", which is incorrect.